cachito/workers/pkg_managers/pip.py

```diff
@@ -17,6 +17,53 @@
 NOTHING = object()  # A None replacement for cases where the distinction is needed
 
 
+def get_pip_metadata(package_dir):
```

Comment:
This will probably need to take an instance of `cachito.workers.scm.SCM` instead, it seems that the name of the cloned directory is not expected to match the repo name (important for the fallback case).

How do want to handle the case where pip has a custom path defined, e.g.
```yaml
packages:
   pip:
     - path: marketplaces/tatooine
     - path: yoda
```

I guess we would use the repo name if the package is at top level and directory name otherwise (again, fallback case)?